src/Foundation/Providers/ArtisanServiceProvider.php

```diff
@@ -12,32 +12,36 @@ class ArtisanServiceProvider extends ArtisanServiceProviderBase
      * @var array
      */
     protected $commands = [
-        'CacheClear'      => 'command.cache.clear',
-        'CacheForget'     => 'command.cache.forget',
-        'ClearCompiled'   => 'command.clear-compiled',
-        'ConfigCache'     => 'command.config.cache',
-        'ConfigClear'     => 'command.config.clear',
-        'Down'            => 'command.down',
-        'Environment'     => 'command.environment',
-        'KeyGenerate'     => 'command.key.generate',
-        'Optimize'        => 'command.optimize',
-        'PackageDiscover' => 'command.package.discover',
-        'QueueFailed'     => 'command.queue.failed',
-        'QueueFlush'      => 'command.queue.flush',
-        'QueueForget'     => 'command.queue.forget',
-        'QueueListen'     => 'command.queue.listen',
-        'QueueRestart'    => 'command.queue.restart',
-        'QueueRetry'      => 'command.queue.retry',
-        'QueueWork'       => 'command.queue.work',
-        'RouteCache'      => 'command.route.cache',
-        'RouteClear'      => 'command.route.clear',
-        'RouteList'       => 'command.route.list',
-        'ScheduleFinish'  => \Illuminate\Console\Scheduling\ScheduleFinishCommand::class,
-        'ScheduleRun'     => \Illuminate\Console\Scheduling\ScheduleRunCommand::class,
-        'Seed'            => 'command.seed',
-        'StorageLink'     => 'command.storage.link',
-        'Up'              => 'command.up',
-        'ViewClear'       => 'command.view.clear',
+        'CacheClear'            => \Illuminate\Cache\Console\ClearCommand::class,
+        'CacheForget'           => \Illuminate\Cache\Console\ForgetCommand::class,
+        'ClearCompiled'         => \Winter\Storm\Foundation\Console\ClearCompiledCommand::class,
+        'ConfigCache'           => \Illuminate\Foundation\Console\ConfigCacheCommand::class,
+        'ConfigClear'           => \Illuminate\Foundation\Console\ConfigClearCommand::class,
+        'Down'                  => \Illuminate\Foundation\Console\DownCommand::class,
+        'Environment'           => \Illuminate\Foundation\Console\EnvironmentCommand::class,
+        'KeyGenerate'           => KeyGenerateCommand::class,
```

Comment:
Use the full namespace here and remove the use statements from the top of the file please.

Reply:
Can do, only reason I didn't was that the class was already being included for use later on in the file, I can swap both usages to a full path tho :)

Reply:
Ah, gotcha. If that's the case then you can leave the import if you want but just make this reference a full path. Up to you, just want full paths here so it's more obvious which are being overridden by Winter.

Reply:
Added full path in b58be85